Mathlib/Data/Set/Insert.lean

```diff
@@ -347,6 +347,10 @@ theorem disjoint_insert_right : Disjoint s (insert a t) ↔ a ∉ s ∧ Disjoint
   obtain ⟨y, hy⟩ := exists_ne x
   exact ⟨y, by simp [hy]⟩
 
+@[simp]
```

Comment:
Could you move the lemma a few lines earlier (before the section heading)? This has nothing to do with compl

Reply:
Moved to `Set/Subsingleton`.